.github/workflows/main.yml

```diff
@@ -108,8 +108,8 @@ jobs:
             -storepasswordincleartext \
             -name "GitHub" \
             -username "$owner" \
-            -password "${{ secrets.GITHUB_TOKEN }}"
-          "$nuget" setapikey "${{ secrets.GITHUB_TOKEN }}" \
+            -password "${{ secrets.NUGET_ACCESS }}"
+          "$nuget" setapikey "${{ secrets.NUGET_ACCESS }}" \
```

Comment:
@fabiangreffrath Something went wrong, no new secret found. Please double check repository settings.

Reply:
That's how it looks:

![Screenshot_20221229_163151_Chrome](https://user-images.githubusercontent.com/6449237/209975194-b90493cf-c4d2-4960-9daf-ea21ad6d4c09.jpg)

Reply:
Secret updated, please try again.

Reply:
I sent you a new PM.

Reply:
![Screenshot_20221229_181843_Chrome](https://user-images.githubusercontent.com/6449237/209987247-73269920-576d-4247-b3ea-957c5082f5f0.jpg)

Reply:
I think we should remove all these new secrets. Sorry for the misunderstanding.